flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+  "automerge-flathubbot-prs": true,
+  "only-arches": ["i386", "x86_64"],
```

Comment:
any reason to be intel only? upstream seems to support more than x86_64

Reply:
I'm not sure. There is an Android CI, but no GNU/Linux ARM one. For me, it looked like less stable than the x86 one. Of course, @flyinghead is the one to answer this.

In my opinion, an ARM CI should be set-up in the upstream repository before including it there.

Reply:
yes, flycast supports armv7 and aarch64 Linux but there is no CI for these currently.

Reply:
Ok I removed the restrictions and opened https://github.com/flyinghead/flycast/issues/223